packages/app/app/components/FavoriteTracksView/index.js

```diff
@@ -58,9 +58,9 @@ const FavoriteTracksView = ({
     });
   };
 
-  const playAll = (tracks) => {
+  const playAll = () => {
     clearQueue();
-    addTracksToQueue(tracks);
+    addTracksToQueue();
```

Comment:
Can you cover this with a test to make sure it actually plays the tracks?
You can do it similarly to the existing playlist view tests.

Reply:
In the original code, the `addTracksToQueue` doesn't use `tracks` from parameter (it uses `tracks` from `props`), also when `playAll` called in `onClick` it doesn't pass any parameter. 
```js
  const addTracksToQueue = () => {
    tracks.map(track => {
      const trackImage = getTrackimage(track);
      addToQueue({
        artist: track.artist.name,
        name: track.name,
        thumbnail: trackImage
      });
    });
  };

  const playAll = (tracks) => {
    clearQueue();
    addTracksToQueue(tracks);
    selectSong(0);
    startPlayback();
  };

  const renderPlayAllButton = () => {
    return (
      <a href='#' className={styles.play_button} onClick={playAll}>
        <Icon name='play'/> Play
      </a>
    );
  };
```